admin/commands/management/remove.go

```diff
@@ -45,6 +49,45 @@ type removeMySQLCommand struct {
 }
 
 func (cmd *removeMySQLCommand) Run() (commands.Result, error) {
+	autoLookup := false
+
+	if cmd.ServiceID == "" && cmd.ServiceName == "" {
+		// Automatic service lookup during removal
+		//
+		// Get services and remove it automatically once it's only one
+		// service registered
+		status, err := agentlocal.GetStatus(agentlocal.DoNotRequestNetworkInfo)
+		if err != nil {
+			return nil, err
+		}
+
+		servicesRes, err := inventoryClient.Default.Services.ListServices(&services.ListServicesParams{
+			Body: services.ListServicesBody{
+				NodeID:      status.NodeID,
+				ServiceType: cmd.serviceType(),
+			},
+			Context: commands.Ctx,
+		})
+		if err != nil {
+			return nil, err
+		}
+		switch {
+		case len(servicesRes.Payload.Mysql) == 1:
+			cmd.ServiceID = servicesRes.Payload.Mysql[0].ServiceID
+		case len(servicesRes.Payload.Mongodb) == 1:
+			cmd.ServiceID = servicesRes.Payload.Mongodb[0].ServiceID
+		case len(servicesRes.Payload.Postgresql) == 1:
+			cmd.ServiceID = servicesRes.Payload.Postgresql[0].ServiceID
+		case len(servicesRes.Payload.Proxysql) == 1:
+			cmd.ServiceID = servicesRes.Payload.Proxysql[0].ServiceID
+		case len(servicesRes.Payload.Haproxy) == 1:
+			cmd.ServiceID = servicesRes.Payload.Haproxy[0].ServiceID
+		case len(servicesRes.Payload.External) == 1:
+			cmd.ServiceID = servicesRes.Payload.External[0].ServiceID
+		}
+		autoLookup = true
```

Comment:
the original suggestion was to return the error message here since we already know that we don't pass service id or service name